package.json

```diff
@@ -1,9 +1,13 @@
 {
   "name": "calculate-4-4-profit",
+  "author": "Cam Hardy",
```

Comment:
really bro, there is a contributors field

Reply:
Well darn you weren't supposed to see that